pkg/cmd/admin/router/router.go

```diff
@@ -293,6 +299,15 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					{Type: dapi.DeploymentTriggerOnConfigChange},
 				},
 				Template: dapi.DeploymentTemplate{
+					Strategy: dapi.DeploymentStrategy{
```

Comment:
@pweil-  On further investigation, 1 is not required - there is already code in https://github.com/ramr/origin/blob/master/pkg/deploy/api/v1/defaults.go#L28  which sets the individual values if they are nil.

So if the router code just does something like:
```   
updatePercent := int(-10)
// snipped  ...   
Strategy: dapi.DeploymentStrategy{  
   Type: dapi.DeploymentStrategyTypeRolling,   
   RollingParams: &dapi.RollingDeploymentStrategyParams{UpdatePercent: &updatePercent}  
}  
// more judicious use of scissors!  ...    
```  then it goes through a conversion (and more) anyway before it gets to the datastore which tacks in the defaults.

See https://gist.github.com/ramr/67005821841097dc2a64 for details - line #s 26-31 specifically.